### PR TITLE
Add preflight check if video is supported by browser

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Description
Enhancement: Show notification if video playback is not possible on public share

Before this PR no error notification was shown if a video can't be played due
to browser incompatibility, for example, mov files on chrome.
(https://stackoverflow.com/questions/28746645)

Now we will show a dedicated notification.
## Related Issue

- Fixes https://github.com/owncloud/enterprise/issues/4632

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [x] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -94,3 +94,7 @@ Options -Indexes
 <IfModule pagespeed_module>
   ModPagespeed Off
 </IfModule>
+#### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
+
+ErrorDocument 403 //core/templates/403.php
+ErrorDocument 404 //core/templates/404.php
```
Comment (phil-davis) [inline .htaccess:100]: accidentally committed

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>apiAuth-mariadb10.2-php7.4</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>apiShareOperationsToShares2-mariadb10.2-php7.4</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>apiShareManagementBasicToShares-mariadb10.2-php7.4</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>apiShareCreateSpecialToShares1-mariadb10.2-php7.4</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>apiCapabilities-mariadb10.2-php7.4</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingNotify-chrome-mariadb10.2-php7.4</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>cliTrashbin-mariadb10.2-php7.4</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>apiProxySmoke-8-8-mariadb10.2-php7.4</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>cliMain-mariadb10.2-php7.4</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>cliExternalStorage-10.6.0-mariadb10.2-php7.4</strong> failed. The build is cancelled...</summary>

**Review by jvillafanez — APPROVED** [review]

**Comment by phil-davis** [comment]
Note: this has missed the branch point of `release-10.8.0` branch that was made last night.
If we want to get it into 10.8 release then either:
- backport to release-10.8.0 branch or
- recreate release-10.8.0 branch from the current tip of core master

**Comment by JanAckermann** [comment]
Rebased by @micbar so this lands in 10.8.0

**Pull request merged**